src/data/items.h

```diff
@@ -2863,8 +2869,8 @@ const struct Item gItemsInfo[] =
 
     [ITEM_FOSSILIZED_FISH] =
     {
-        .name = _("FosslzedFish"),
-        .pluralName = _("FosslzedFishes"),
+        .name = HANDLE_EXPANDED_ITEM_NAME("FosslzedFish", "Fossilized Fish"),
+        .pluralName = HANDLE_EXPANDED_ITEM_NAME("FosslzedFishes", "Fossilized Fishes"),
```

Comment:
Isn't the plural to Fish also Fish?

Reply:
I just copied what was already there! iirc, "fishes" means "many different species of fish", whereas plural "fish" is "many of the same species of fish". I'm not sure which is applicable here though.